src/apiMiddleware/config.js

```diff
@@ -0,0 +1,3 @@
+export const LOGIN_URL = 'http://localhost:3002/user/login';
```

Comment:
The base path (`http://localhost:3002`) is duplicated. Let's define here only the endpoints and create the URL at the query level.